package.json

```diff
@@ -69,7 +69,9 @@
     "@frctl/handlebars": "^1.2.15",
     "@frctl/mandelbrot": "^1.10.1",
     "minimist": "^1.2.7",
-    "npm-watch": "^0.11.0"
+    "npm-watch": "^0.11.0",
+    "postcss": "^8.4.38",
+    "postcss-mixins": "^10.0.0"
```

Comment:
You asked me about where to put these. I gave you an incorrect answer. Please move to `peerDependencies`. Manually doing this will be sufficient for testing, but uninstalling and re-installing via appropriate commands should be done before merge.